client/components/LoadUser.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect } from 'react';
+import { useAuth } from '../store/users/AuthContext';
+import { useAPI } from '../api/api';
+
+export const LoadUser = () => {
+  const { currentUser } = useAuth();
+  const api = useAPI();
+
+  const getUser = () => {
+    try {
+      api.get(`/users/${currentUser.uid}`).then(({ data }) => {
+        localStorage.setItem('picture', data.picture);
+        localStorage.setItem('username', data.username);
+        localStorage.setItem('email', data.email);
+      });
```

Comment:
Strong recommendation to use Context/Redux here

Reply:
You know I actually did that and got rid of it. So you mean to have like a `userContext.jsx` file with a provider that wraps the components it affects as well as a `useAuth`? Like we have for `authContext.jsx`?